dictionaries/en_US/src/additional_words.txt

```diff
@@ -56,12 +56,31 @@ rebrand
 rebranding
 teleconverter
 tricep
+unaggregated
```

Comment:
Please add these words (all the ones you added) to [coding-terms](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/software-terms/src/coding-terms.txt).

This dictionary is for US English words, general computing/coding terms are better of in the software terms dictionary. That way, those that do not use US English will also get the benefit.